Update debugging info in README.md (#30)
temporalio/vscode-debugger-plugin: README.md

- Install dependencies `npm ci`
- Build the extension and WebView: `npm run build`
- Watch filesystem and rebuild on change: `npm run build.watch` (recommended)

### Debugging with VSCode

> - To launch the extension a [launch.json](./.vscode/launch.json) is provided that will run the extension in a new VS Code window (see [Run and Debug view](https://code.visualstudio.com/docs/editor/debugging#_run-and-debug-view)).

Reviewer: > that will run the extension

when you do what with the file?

> - To launch the extension a [launch.json](./.vscode/launch.json) is provided that will run the extension in a new VS Code window (see [Run and Debug view](https://code.visualstudio.com/docs/editor/debugging#_run-and-debug-view)).

Reviewer 2: ```suggestion
- Use the provided [launch configuration](./.vscode/launch.json) to run the extension in a new VS Code window (see [Run and Debug view](https://code.visualstudio.com/docs/editor/debugging#_run-and-debug-view)).
```
